pattoo/db/schema/chart_datapoint.py

```diff
@@ -42,6 +44,11 @@ class Meta:
         interfaces = (graphene.relay.Node,)
 
 
+class ProtectedChartDataPoint(graphene.Union):
```

Comment:
Add a docstring to explain what you are trying to do here. It will help with future maintenance.